### PR TITLE
714 fix bug with elasticsearch connector and click analytics

### PR DESCRIPTION
**Pull request by joemcelroy**
fixes https://github.com/elastic/search-ui/issues/714

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _search-ui_ successfully!

* [View deploy preview](https://search-2mnvhysti-elastic-dev.vercel.app)

* Built with commit b58c0f4da29f73f73e779c1181c4a921c6b8c828

> Issues? Visit #next-docs in Slack

File: packages/search-ui-elasticsearch-connector/src/handlers/common.ts [hunk 1]
```diff
@@ -20,7 +20,10 @@ export function fieldResponseMapper(
       };
     },
     {
-      id: { raw: item.id }
+      id: { raw: item.id },
+      _meta: {
+        id: item.rawHit?._id
```
Comment (yakhinvadim) [inline packages/search-ui-elasticsearch-connector/src/handlers/common.ts:25]: Looking at optional chaining... So it's possible for `_meta.id` to be undefined?
Will we hit the same error if so?

File: packages/search-ui-elasticsearch-connector/src/handlers/search/Configuration.ts [hunk 2]
```diff
@@ -18,13 +18,7 @@ export function getResultFields(
...
   highlightFields: string[];
 } {
-  const hitFields = Object.keys(resultFields).reduce((sum, fieldKey) => {
-    const fieldConfiguration = resultFields[fieldKey];
-    if (fieldConfiguration.raw) {
-      sum.push(fieldKey);
-    }
-    return sum;
-  }, []);
+  const hitFields = Object.keys(resultFields);
```
Comment (joemcelroy) [inline packages/search-ui-elasticsearch-connector/src/handlers/search/Configuration.ts:21]: this is unrelated to the bug but required from the bump of searchkit. Found an issue where we need all fields (raw and highlighted) to be hitFields.

**Review by yakhinvadim — APPROVED** [review]
LGTM, thanks for covering my miss in WS connector. Just 1 question:

**Pull request merged**